src/render/decorations/IHyprWindowDecoration.hpp

```diff
@@ -39,6 +39,12 @@ class IHyprWindowDecoration {
 
     virtual bool                     allowsInput();
 
+    virtual void                     dragWindowToDecoration(CWindow*, const Vector2D&);
```

Comment:
same. begin? end?

And generally all 3 should be prefixed with `on` as they are callbacks